rl_bakery/data_manager/builder.py

```diff
@@ -0,0 +1,19 @@
+from rl_bakery.data_manager.data_manager import DataManager, DATANAME, InMemoryStorage, TFAgentStorage
+
+
+def build_inmemory_data_manager(application):
```

Comment:
Should we create a builder/factory to take care of the creation of the data_manager. That way it can support other data storage such as s3, etc...

Reply:
yeah I agree, we should add a filesystem builder (which would work for hdfs, s3, gcp etc.)